web/src/components/layout/layout.tsx

```diff
@@ -248,10 +248,7 @@ class Layout extends React.PureComponent<LayoutProps, LayoutState> {
       featureStorageKey,
```

Comment:
Because we no longer redirect to `/404` we can't use pathnames to determine classNames